libs/getdns/Makefile

```diff
@@ -49,16 +48,15 @@ define Package/getdns/config
 	source "$(SOURCE)/Config.in"
 endef
 
-CONFIGURE_ARGS += \
-		$(if $(CONFIG_GETDNS_ENABLE_STUB_ONLY), --enable-stub-only, ) \
-		--without-libidn \
-		$(if $(CONFIG_GETDNS_ENABLE_IDN_LIBIDN2), , --without-libidn2 ) \
-		--with-ssl="$(STAGING_DIR)/usr" \
+# CMAKE options for GetDNS are described in this document:
+# https://getdnsapi.net/quick-start/cmake-quick-start/
+CMAKE_OPTIONS += -DBUILD_LIBEV=OFF
+CMAKE_OPTIONS += -DBUILD_LIBEVENT2=OFF
+CMAKE_OPTIONS += -DBUILD_LIBUV=OFF
+
+CMAKE_OPTIONS += -DENABLE_STUB_ONLY=$(if $(CONFIG_GETDNS_ENABLE_STUB_ONLY),ON,OFF)
+CMAKE_OPTIONS += -DUSE_LIBIDN2=$(if $(CONFIG_GETDNS_ENABLE_IDN_LIBIDN2),ON,OFF)
 
-# This will make 'configure' think that our libbsd.so is missing the
-# functions inet_pton, inet_ntop, strlcpy and use the builtin. This
-# removes the libbsd dependency
-CONFIGURE_VARS += LIBBSD_LIBS=-lc
```

Comment:
Is this still an issue?

Passing -DCMAKE_DISABLE_FIND_PACKAGE_BSD=ON should fix it if so.

Reply:
Have added your suggestion, thanks.

Reply:
Do note that I have not tested if this is a problem.